diff -Naur xmlrpc-c-1.06.38_orig/ltconfig xmlrpc-c-1.06.38/ltconfig
--- xmlrpc-c-1.06.38_orig/ltconfig	2004-10-21 01:45:14.000000000 +0200
+++ xmlrpc-c-1.06.38/ltconfig	2010-03-03 11:39:16.147341334 +0100
@@ -1226,8 +1226,8 @@
 
   *)
     if $LD --help 2>&1 | egrep ': supported targets:.* elf' > /dev/null; then
-      archive_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname -o $lib'
-      archive_expsym_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}-soname $wl$soname ${wl}-retain-symbols-file $wl$export_symbols -o $lib'
+      archive_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}--hash-style=gnu ${wl}-soname $wl$soname -o $lib'
+      archive_expsym_cmds='$CC -shared $libobjs $deplibs $linkopts ${wl}--hash-style=gnu ${wl}-soname $wl$soname ${wl}-retain-symbols-file $wl$export_symbols -o $lib'
     else
       ld_shlibs=no
     fi
